File: CodeFreak1/ClientApp/src/app/event/dtos/event-user-view-model.ts
import { RequestStatus } from "../../Security/Dtos/request-status";
import { UsersViewModel } from "../../Security/Dtos/users-view-model";
import { EventViewModel } from "./event-view-model";
import { RolesViewModel } from "../../Security/Dtos/roles-view-model";
import { FileViewModel } from "../../Security/Dtos/file-view-model";

export class EventUserViewModel extends RequestStatus{
  public User: UsersViewModel;
  public Event: EventViewModel;
  public Roles: Array<RolesViewModel>;
  public UserImage: FileViewModel;
}

File: CodeFreak1/ViewModel/EventUserViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CodeFreak1.ViewModel
{
    public class EventUserViewModel:RequestStatus
    {
        public EventViewModel Event { get; set; }
        public UsersViewModel User { get; set; }
    }
}
